### ISSUE TITLE
Private message are not saved - Setting is on !

### ISSUE THREAD

**Issue by JoakimFalk** [PHPFusion/PHPFusion#310]
![setting-on](https://cloud.githubusercontent.com/assets/4210004/9897878/d430c400-5c4d-11e5-907c-1cfa079b0a63.png)

**Comment by meangczac** [comment]
Super Admin message were not saved. Members were.

**Comment by JoakimFalk** [comment]
Nor member or admins works on my test. tried with system setting, default ( to read it system setting and yes save messages setting with ssystem on )

**Comment by meangczac** [comment]
I'm working on it now

**Comment by meangczac** [comment]
Ok, I found the problem. Outbox initialisation is bad. I'll double check before I git.

I also found in v7 codes that if outbox limit is at max, the PM have a feature to attempt to salvage space automatically removing the oldest message you sent to the current sender.
https://github.com/php-fusion/PHP-Fusion/blob/7.02.07/messages.php#L200-L204
And it will not save message at all if outbox is full nor does it prevent you from sending message out.
It's not consistent to the rules if one wants to nit pick at it.

Just to inform. Don't want later say it's a bug. I follow v7.
